.rubocop.yml

```diff
@@ -62,7 +62,7 @@ CyclomaticComplexity:
 Delegate:
   Enabled: false
 
-DeprecatedHashMethods:
+PreferredHashMethods:
   Enabled: false
 
 Documentation:
```

Comment:
![You did the thing.](http://i0.kym-cdn.com/photos/images/original/000/840/380/c13.gif)